src/main/java/training/Employees.java

```diff
@@ -0,0 +1,44 @@
+package training;
+
+import java.io.FileWriter;
+import java.io.IOException;
+
+import org.json.simple.JSONArray;

```

Comment:
Which package you are using? Is it being included in the gradle file?

Reply:
no its not in gradle file actually i have imported it from intellij lib

Reply:
What will be the problem if you do not addd in gradle file? Can you guess?

Reply:
junit was also not present in gradle file. I added junit 4 from intellij lib. I'm not familiar of gradle, so i'm not knowing what problem we will face if we do not add in gradle file. i searched alot on google about how to add jackson dependency in build gradle project all got is [  import com.fasterxml.jackson.datatype; ] and i can use this, only if i'm adding jackson dependency from intellij lib